ruby-kafka.gemspec

```diff
@@ -37,7 +37,7 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "snappy"
   spec.add_development_dependency "colored"
   spec.add_development_dependency "rspec_junit_formatter", "0.2.2"
-  spec.add_development_dependency "dogstatsd-ruby", ">= 3.0.0"
+  spec.add_development_dependency "dogstatsd-ruby", "~> 3.0"
```

Comment:
Core is using `dogstatsd-ruby (4.0.0)`. Does it mean it would conflict?

Reply:
I just tested it out locally and seems like `development_dependency` is not pulled in Gemfile.lock and has no impact.
This is because datadog part of this gem is not yet used in Shopify
From readme on `0-4-3-with-exit_if` branch

```
# This enables the reporter:
require "kafka/datadog"

# Default is "ruby_kafka".
Kafka::Datadog.namespace = "custom-namespace"

# Default is "127.0.0.1".
Kafka::Datadog.host = "statsd.something.com"

# Default is 8125.
Kafka::Datadog.port = 1234
```

Which we don't do in Shopify. So all good.